src/components/TextArea.js

```diff
@@ -54,7 +54,9 @@ const TextArea = React.createClass({
       >
         <textarea
           {...elementProps}
-          ref='textarea'
+          ref={ref => {
+            this.textarea = ref;
+          }}
```

Comment:
`this.refs.textarea` no longer works and throws a console error. It may have to do with the latest version of React.